src/phoenix/core/traces.py

```diff
@@ -47,16 +47,10 @@ def get_projects(self) -> Iterator[Tuple[int, str, "Project"]]:
 
     def archive_project(self, id: int) -> Optional["Project"]:
```

Comment:
check for 0 instead, because 0 is always the default project
```suggestion
    def archive_project(self, id: int) -> Optional["Project"]:
        if id == 0:
            return None
```

Reply:
addressed in 530e6af9a7ac5a01f2c09905d09922303e70f020